02-introduction.rst

```diff
@@ -10,7 +10,8 @@ and your new friends (among others) are named "vectors", "arrays", "views" or
 Simple example
 --------------
 
-Let's take a very simple example, random walk.
+Let's take a very simple example, random walk. (You can execute any code below
+from the `code` folder, ideally inside an IPython session or Jupyter notebook.)
```

Comment:
I see a potential for confusion since `timeit` is also in the standard-library and there is an IPython/Jupyter `%timeit` magic function, but maybe that's ok. Apart from that, I just feel a short notice would be ok about going to the `code` folder the first time, before the reader wonders where the `tools` import comes from.